cms/templates/cms/home_page.html

```diff
@@ -75,7 +75,46 @@
         </div>
       </div>
 
-            <section class="row current-programs-section">
+      <div class="row how-it-works-section">
```

Comment:
This is what we had before... can you create a new issue for this, as it is a semantic/accessibility issue, and I want to get this change out as soon as possible.